eng/Versions.props

```diff
@@ -74,7 +74,7 @@
     <MicrosoftVisualStudioPackagesVersion>17.5.274-preview</MicrosoftVisualStudioPackagesVersion>
     <RoslynPackageVersion>4.6.0-3.23164.6</RoslynPackageVersion>
     <VisualStudioLanguageServerProtocolVersion>17.6.4-preview</VisualStudioLanguageServerProtocolVersion>
-    <MicrosoftNetCompilersToolsetVersion>4.6.0-2.23128.3</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>$(RoslynPackageVersion)</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This change does not look correct. What's the purpose of it?

Reply:
I'm not sure if this change should be conditioned for source-build only. It makes sense for `Microsoft.Net.Compilers.Toolset` package to be consumed from the same build as other `roslyn` packages, as it is built in `roslyn`. However, I don't know if this was an omission or intentional.

Reply:
Source-build consumes packages from other repos, via intermediate packages, therefore we require that all packages from one repo have a matching version.